server/src/test/java/io/crate/integrationtests/AlterTableIntegrationTest.java

```diff
@@ -25,31 +25,38 @@
 import static io.crate.testing.Asserts.assertThrowsMatches;
 import static io.crate.testing.SQLErrorMatcher.isSQLError;
 import static io.netty.handler.codec.http.HttpResponseStatus.BAD_REQUEST;
-import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.containsString;
 import static org.hamcrest.Matchers.startsWith;
 
 import org.junit.Test;
 
-
 public class AlterTableIntegrationTest extends SQLIntegrationTestCase {
 
     @Test
-    public void test_alter_soft_delete_setting_for_partitioned_tables() {
-        execute("create table test(i int) partitioned by (i)");
-        assertThat(response.rowCount(), is(1L));
-        execute("alter table test set (\"soft_deletes.enabled\" = false)");
```

Comment:
`alter final setting` used to pass - on empty open table (which is the case here), now it fails and I had to rewrite it to create with - I think it's fine as it's aligned better with the error message _"Creating tables with soft-deletes disabled is no longer supported"_